scripts/owners.json

```diff
@@ -0,0 +1,74 @@
+[
+  {
+    "teams": [
+      "core",
+      "ui"
+    ],
+    "modules": [
+      "examples",
+      "app-tests-wrapper",
+      "instrumentation-tests",
+      "qa-test-app",
+      "android-auto-app",
+      "libtesting-thirdparty",
+      "libtesting-navigation-base",
+      "libtesting-navigation-util",
+      "libtesting-utils"
+    ]
+  },
+  {
+    "teams": [
+      "core"
+    ],
+    "modules": [
+      "libnavigation-android",
+      "libnavigator",
+      "libnavigation-base",
+      "libnavigation-router",
+      "libnavigation-metrics",
+      "libtrip-notification",
+      "libnavigation-util",
+      "libnavigation-core"
+    ]
+  },
+  {
+    "teams": [
+      "app"
+    ],
+    "modules": [
+      "libnavigation-copilot"
+    ]
+  },
```

Comment:
Does this mean I will miss all of the PRs except for Copilot ones? Not sure I like this idea, I often review UI PRs. Also it is useful for me to be aware of any changes in Nav SDK, so that I will be ready to integrate them in 1TAP.

Reply:
You can setup notifications via [github notifications](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications#configuring-your-watch-settings-for-an-individual-repository). You shouldn't be attached as a reviewer to the PR for that